lib/Db/PersonMapper.php

```diff
@@ -297,16 +297,26 @@ public function removeIfEmpty(int $personId) {
 	 */
 	public function deleteOrphaned(string $userId): int {
 		$sub = $this->db->getQueryBuilder();
-		$sub->select(new Literal('1'));
+		$sub->select('*');
```

Comment:
any reason why you needed "*" here? you should keep "1" unless there is reason not to.

Reply:
No reason ... it was just one of many tests. :sweat_smile: